frontend/src/views/Home.vue

```diff
@@ -86,27 +86,39 @@
     </v-container>
 
     <!-- Join Us Section -->
-    <v-container>
+    <v-container id="join" ref="joinus">
       <h1>
         Join Us
       </h1>
-      <h3>
+      <p>
         Find our stall at O-week, or just turn up an event and get to know us!
-      </h3>
+      </p>
       <h2>
         Join our community online
       </h2>
+      <div id="joinImg" >
+        <a href="https://www.facebook.com/groups/csesoc">
+          <img src="@/assets/facebook_logo.png" class="zoom">
+        </a>
+        <a href="https://csesoc-community.slack.com/">
+          <img src="@/assets/slack_logo.png" class="zoom">
+        </a>
+        <!-- need to add in the csesoc discord link-->
+        <a href="https://www.facebook.com/groups/csesoc">
+          <img src="@/assets/discord_logo.png" class="zoom">
+        </a>
+      </div>
```

Comment:
Is it valuable to have this as its own component?

Reply:
yeah i think ill do that